Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -172,7 +172,7 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
     if (placeholderImage) {
         [view pin_setPlaceholderWithImage:placeholderImage];
     }
-    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsNone;
+    PINRemoteImageManagerDownloadOptions options = [view pin_defaultOptions];
```

Comment:
I'm finding it quite hard to come up with a good reason! I guess its personal preference. For example, I wouldn't expect people to create categories that conformed to `PINRemoteImageCategory`. I find that if you have the need for that, then make a PR. If the extension brings in other dependencies, then perhaps create a subpod for it along with your feature <cough> ASImageNode <cough> ;). The current way, it just means that if there's a situation where you _expect_ every category to implement a method, its either a major version bump or another `respondsToSelector` wrap-around.

I've made the changes to make it `@optional`. Thanks! :)